### PR TITLE
docs: update readme - add fund recovery tutorial

### PR DESCRIPTION
**Pull request by khanti42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,110 @@
...
-### UI
+#### UI
+
 ```bash
 # Running Wallet UI
 yarn workspace wallet-ui start
 ```
+
 ```bash
 # Running Storybook
```
Comment (stanleyyconsensys) [inline README.md:109]: i think we can remove story book, as we are no longer support

File: README.md [hunk 2]
```diff
@@ -62,45 +114,55 @@ yarn workspace wallet-ui storybook
 - Wallet UI dapp: http://localhost:3000/
 - Storybook: http://localhost:6006/
 
-# Dapp intergation Guide
+## Dapp Integration Guide
+
+This guide is for a dApp that would be compatible only with MetaMask. To create a dApp compatible with all wallets in the Starknet ecosystem, prefer the get-starknet library. A sample dApp can be found [here](https://github.com/PhilippeR26/Cairo1JS/tree/main/src). If you still want to directly interact with the snap follow the tutorial below:
```
Comment (stanleyyconsensys) [inline README.md:119]: i think we dont need to refer to external dapp? as the current dapp in this repo already explain how to do Integration?

**Review by stanleyyconsensys — CHANGES_REQUESTED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (khanti42) [reply README.md:119]: It's just that in the current README.md we don't explain how to do multi-wallet compatible dApp on starknet. To do so dev need to use get-starknet official repo and not direct call to the snap. Calling the snap directly as described in the README.md shows how to do a dApp on starknet compatible with `metamask only`. 

The goal here is to clearly tell to devs what is the best dev strategy in terms of wallet interoperability with the starknet ecosystem. 

wdyt ?

**Review by stanleyyconsensys — APPROVED** [review]
LGTM

**Pull request merged**